Automattic/wp-calypso
# Checklist: add vertical-specific tasks

#### Changes proposed in this Pull Request

* As a part of new onboarding project, this PR add new vertical specific checklist tasks so the users will see personalized tasks based on their site vertical. See p5XAZ9-24V-p2 for more details.

#### Testing instructions



* Apply D23270-code to your sandboxed `public-api.wordpress.com`.
* Create a new `business` website though the new onboarding signup flow.
* Move to the checklist page and see if the checklist shows vertical specific tasks. For example, `Music` website should show the `Add your services` item.
* ~Only the two following tasks are currently working: `Edit about text` and `Change homepage photo`~. All vertical tasks are available now.
* The flow of new vertical tasks would be like as follow:
<img width="1270" alt="screen_shot_2019-01-18_at_10 50 46" src="https://user-images.githubusercontent.com/212034/51755152-4bafb100-2101-11e9-83fe-78a8a2437a83.png">

Demo:
![kapture 2019-01-26 at 0 58 17](https://user-images.githubusercontent.com/212034/51756965-8ddaf180-2105-11e9-914c-811beda93b87.gif)
